index.ts

```diff
@@ -93,7 +97,7 @@ class Money {
         return new Money(amount, currency)
     }
 
-    static fromDecimal(amount: number|any, currency: string|any, rounder?: string|Function): Money {
+    static fromDecimal(amount: number | Partial<IMoney>, currency: string|any, rounder?: string|Function): Money {
```

Comment:
I would probably use IMoney directly since none of its properties are optional for this function.

```suggestion
    static fromDecimal(amount: number | IMoney, currency: string|any, rounder?: string|Function): Money {
```